### PR TITLE
HMAC Push Notification Polish

### PR DESCRIPTION
**Pull request by nplasterer**
This polishes up the readme and the example in prep for the group chat push work.

### CODE REVIEW

File: library/src/main/java/org/xmtp/android/library/push/README.md [hunk 1]
```diff
@@ -77,7 +81,24 @@ This example branch can serve as the basis for what you might want to provide fo
     ```
 
     ```kotlin
-    XMTPPush(context, "10.0.2.2:8080").subscribe(conversations.map { it.topic })
+    val subscriptions = conversations.map {
+        val hmacKeysResult = ClientManager.client.conversations.getHmacKeys()
+        val hmacKeys = hmacKeysResult.hmacKeysMap
```
Comment (alexrisch) [inline library/src/main/java/org/xmtp/android/library/push/README.md:86]: Want to move this out of subscriptions map?

Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/push/README.md:86]:
```suggestion
    val hmacKeys = hmacKeysResult.hmacKeysMap
    val subscriptions = conversations.map {
        val hmacKeysResult = ClientManager.client.conversations.getHmacKeys()
```

File: library/src/main/java/org/xmtp/android/library/push/README.md [hunk 2]
```diff
@@ -77,7 +81,24 @@ This example branch can serve as the basis for what you might want to provide fo
     ```
 
     ```kotlin
-    XMTPPush(context, "10.0.2.2:8080").subscribe(conversations.map { it.topic })
+    val hmacKeys = hmacKeysResult.hmacKeysMap
+    val subscriptions = conversations.map {
+        val hmacKeysResult = ClientManager.client.conversations.getHmacKeys()
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/push/README.md:86]:
```suggestion
    val hmacKeysResult = ClientManager.client.conversations.getHmacKeys()
    val subscriptions = conversations.map {
        val hmacKeys = hmacKeysResult.hmacKeysMap
```

File: library/src/main/java/org/xmtp/android/library/push/README.md (hunk 1 above)
Comment (nplasterer) [reply library/src/main/java/org/xmtp/android/library/push/README.md:86]: Good call 👍  Moved results out like the example has.

**Review by alexrisch — APPROVED** [review]

**Pull request merged**